worker/dbaccessor/tracker.go

```diff
@@ -81,6 +81,12 @@ type trackedDBWorker struct {
 	metrics *Collector
 
 	pingDBFunc func(context.Context, *sql.DB) error
+
+	// Metrics for the engine report.
+	pingDuration    time.Duration
```

Comment:
If I was being super clean when doing this, I would make a struct type with these members and just delegate the report method to it..